test/XRP/xrp-protocol-buffer-conversion.test.ts

```diff
@@ -1,3 +1,8 @@
+/* eslint-disable @typescript-eslint/no-non-null-asserted-optional-chain --
+ * This rule should never be disabled in production, but for tests,
+ * having a x?.y?.z! chain is a succinct way of getting the typing correct.
+ */
```

Comment:
I really don't know about disabling this rule. I wonder if some constructors need to have their types updated?

An example from line 211 would be:
```ts
XRPCurrencyAmount.from(
  testPaymentProtoAllFieldsSet.getAmount()?.getValue()!,	        
)
```

Reply:
I think this is reasonable to do. It's only test code and the protocol buffers tend to be pretty verbose if you do it the 'right' way.